### ISSUE TITLE
How to undo the changes of Lick

### ISSUE THREAD

**Issue by PhilippPro** [noryb009/lick#38]
I tried it Lick with Puppy Linux and it was great. 
Now I cleaned up again, but I cannot enable "fast Windows start" again. What do I have to do to enable it again? The option is not visible anymore. 

![fail](https://user-images.githubusercontent.com/11573242/106354077-7edc3a80-62ef-11eb-942f-f3b57abb8204.PNG)

**Comment by PhilippPro** [comment]
I changed it manually with this: https://superuser.com/questions/656459/disabling-hybrid-boot-in-windows-8/879674#879674

**Comment by noryb009** [comment]
@PhilippPro Good point, the program should restore the initial fastboot config when the bootloader is uninstalled. I'll treat this as a feature request.
